server/filestore.go

```diff
@@ -503,7 +503,11 @@ func newFileStoreWithCreated(fcfg FileStoreConfig, cfg StreamConfig, created tim
 	// Also make sure we get rid of old idx and fss files on return.
 	// Do this in separate go routine vs inline and at end of processing.
 	defer func() {
-		go fs.cleanupOldMeta()
```

Comment:
Probably do check before spinning go routine.

Reply:
`if fs != nil { go fs.cleanupOldMeta() }`

Reply:
sounds good, updated to use that instead.